hyfetch/distros/rhino.py

```diff
@@ -0,0 +1,28 @@
+
+from . import AsciiArt
+
+   Rhino Linux = AsciiArt(match=r'''"Rhino Linux"* | "Rhino"*''', color='5 4', ascii=r"""
+
```

Comment:
Uhhhhhhhhh I'm pretty sure this isn't valid python 😭 

The py files should be auto-generated